app/routes/manual-ledger-check.js

```diff
@@ -40,9 +40,7 @@ module.exports = [{
       query: Joi.object({
         paymentRequestId: Joi.number().required(),
         'ar-value': Joi.number().required(),
-        'ap-value': Joi.number().required(),
-        'ap-percentage': Joi.number().required(),
-        'ar-percentage': Joi.number().required()
+        'ap-value': Joi.number().required()
       }).options({ allowUnknown: true }),

```

Comment:
Could, and possibly should, be extracted into its own file à la our payload schemas and imported in to keep the functionality sep. and allow for greater control over expected error messages etc -- I would say this is Out of scope (OOS) for this ticket but could you add this as a tech debt ticket in our backlog?